modules/react/button/lib/ExternalHyperlink.tsx

```diff
@@ -1,8 +1,10 @@
 import React from 'react';
-import {styled, createComponent, StyledType} from '@workday/canvas-kit-react/common';
+import {createComponent} from '@workday/canvas-kit-react/common';
 import {extLinkIcon} from '@workday/canvas-system-icons-web';
-import {SystemIcon, systemIconStyles} from '@workday/canvas-kit-react/icon';
-import {Hyperlink, HyperlinkProps} from './Hyperlink';
+import {SystemIcon, systemIconStencil} from '@workday/canvas-kit-react/icon';
+import {HyperlinkProps, hyperlinkStencil} from './Hyperlink';
```

Comment:
i think this import is ok since they're both under @workday/canvas-kit-react/button` and I don't want a circular dependency

Reply:
Correct. You should always use relative imports within the same slash import. Think of it like this "All imports should be relative unless you need to use `lib/`". Or "All imports within the same 'module' should be relative"